Code/ObjectMapping/RKObjectMapping.h

```diff
@@ -68,6 +68,8 @@
  */
 @interface RKObjectMapping : RKMapping <NSCopying>
 
+- (void)copyPropertiesFromMapping:(RKObjectMapping *)mapping;
```

Comment:
could you please add documentation for this method, and move the declaration to the appropriate section of the header?